backend/typescript/services/implementations/camperService.ts

```diff
@@ -167,6 +167,41 @@ class CamperService implements ICamperService {
 
     return camperDtos;
   }
+
+  async getCamperByChargeId(chargeId: number): Promise<CamperDTO> {
+    try {
+      // eslint-disable-next-line prettier/prettier
+      const camper: Camper | null = await MgCamper.findOne({ "chargeId": chargeId });
```

Comment:
there can be multiple campers with the same chargeId and we should be returning all of them